docs/contribute/e2eTesting.md

```diff
@@ -28,7 +28,10 @@ For more infos on Protractor see [Protractor API documentation](http://angular.g
   ```
   node ./node_modules/protractor/bin/webdriver-manager update
   ```
```

Comment:
Mach ich. Ich verstehe es auch nicht. Aber ich habe zum wiederholten Male Zeit investiert für den Setup, den ich lieber in Scenarioo investiert hätte.